### ISSUE TITLE
PHP 7.4 Support

### ISSUE THREAD

**Issue by VarroReve** [postaddictme/instagram-php-scraper#576]
When running in PHP 7.4, it will throw a notice level error, in file `raiym/instagram-php-scraper/src/InstagramScraper/Model/Media.php` line 542:

```
Trying to access array offset on value of type null
```

In PHP 7.3, that's right:

```
$foo = null;
$bar = $foo['bar']; // $bar is null
```

But not for PHP 7.4.

**Comment by kikemarto** [comment]
How did you solve? @VarroReve

**Comment by Charmer** [comment]
> 
> 
> How did you solve? @VarroReve

Looks like it had been solved in #610
